debian/control

```diff
@@ -931,12 +931,24 @@ Replaces: mariadb-connect-engine-10.0,
           mariadb-connect-engine-10.2,
           mariadb-connect-engine-10.3,
           mariadb-connect-engine-10.4
+Suggests: mariadb-plugin-connect-jdbc
 Description: Connect storage engine for MariaDB
  Connect engine supports a number of file formats (dbf, xml, txt, bin, etc),
  connections to ODBC tables and remote MySQL tables, as well as a number of
  other interesting features.
  This package contains the Connect plugin for MariaDB.
 
+Package: mariadb-plugin-connect-jdbc
+Architecture: any
+Depends: mariadb-plugin-connect,
+         default-jdk,
```

Comment:
We can not use  `jre` instead of `jdk`, because we are compiling java source and need `jar`, that is part of `jdk` and not `jre`.

Reply:
`Build-Depends` != `Depends:`.

`Build-Depends: default-jdk` of entire mariadb. This is where dependences required to build mariadbar ethere.

`Depends: default-jre-headless`, is what is required at runtime, because `jni.h` and `jar` aren't needed at runtime.

In the same way that `libxml2-dev` is a build depends and `libxml2` is a Depends of `mariadb-plugin-connect`.